### ISSUE TITLE
Desktop v2.1.5 sync fail "json.decoder.JSONDecodeError

### ISSUE THREAD

**Issue by huajiaowei** [tsudoko/anki-sync-server#8]
Thanks for you work,I configured server work fine.
But when I sync from anki desktop to server,It dumps

Traceback (most recent call last):
  File "aqt\sync.py", line 382, in _sync
  File "aqt\sync.py", line 362, in _abortingSync
  File "anki\sync.py", line 41, in sync
  File "anki\sync.py", line 593, in meta
  File "json\__init__.py", line 354, in loads
  File "json\decoder.py", line 339, in decode
  File "json\decoder.py", line 357, in raw_decode
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

============
anki desktop:
 version 2.1.5
 platform windows 10

**Comment by wyrainer** [comment]
My anki is 2.1.3, it is ok . 
I think it is not compatible with v2.1.5 maybe

**Comment by tsudoko** [comment]
Can't reproduce on 2.1.7 or 2.1.5. Are you seeing anything in the server log?

**Comment by TimDumol** [comment]
I experienced the same issue trying to access from Anki  2.1.3. I used a packet sniffer and noticed that the requests were going to `/6sync/` instead of `/sync/`. It seems to be related to the `hostNum` change mentioned [here](https://github.com/ankidroid/Anki-Android/issues/4921).

This seems to happen when you are currently synced to AnkiWeb, and then switch to ankisyncd. If you create a new profile and use ankisyncd from the start, this doesn't seem to be an issue. This issue only occurs when using Anki Desktop -- AnkiDroid doesn't use the `${hostnum}sync` URLs yet, it seems. As a workaround, I uploaded using AnkiDroid and then created a new Anki Desktop profile, and downloaded from there.

The fix for this is probably as simple as making `/\d+sync/` and `/\d+msync/` aliases for `/sync` and `/msync`.
